src/extension/utils.js

```diff
@@ -703,7 +703,15 @@ export async function updateProjectAuthorizationHeaderRules() {
         condition: {
           regexFilter: `^https://[a-z0-9-]+--${repo}--${owner}.aem.(page|live)/.*`,
           requestMethods: ['get', 'post'],
-          resourceTypes: ['main_frame'],
+          resourceTypes: [
+            'main_frame',
+            'script',
+            'stylesheet',
+            'image',
+            'xmlhttprequest',
+            'media',
+            'font',
```

Comment:
One more item that could be added is `sub_frame`, but I was a bit unsure, because I didn't really think of a case when I used it.
Normally, if I need to display another document I just use the `fragment` block, but if anybody thinks it makes sense, I can add it.

Reply:
I guess this is the result of testing this feature on a site with no css or js ;)

Thanks @andreituicu. One thing to keep in mind, some of this will change with #788. Instead of setting the token in the options page it will be provided by the admin API.

Reply:
@andreituicu Yeah, you got it. Just making you aware (in case you didn't see it) that the way you specifying the auth token via the options page will go away with #788. Your change is still valid for that PR.

Reply:
@andreituicu Good job! I'll make sure #788 is updated accordingly.